packages/payment-detection/test/erc777/mocks.ts

```diff
@@ -181,18 +181,66 @@ const mockFlows = [
     },
   ],
 ];
+const mockOneOff = [
```

Comment:
I think the proper name for this file should be `fixtures` no?

Reply:
Hmm the others constants in the file are named `mockXXX` as well.